packages/docs/docs/tutorial/integrate-sdk/ios.mdx

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_label: iOS
+---
+
+import Step from '@components/Step';
```

Comment:
It seems that this `Step` is unused.

Reply:
Yes, we talked about it last night and we decided to separate the usages in AC and docs apart, which means AC will not try to re-use the mdx files in docs.
Hence the `Step` should be removed from this docs mdx file, and I will make a copy of this in AC but with `Step`